internal/proxy/hook/authz/authz.go

```diff
@@ -177,7 +185,7 @@ func (a Authz) ServeHook(res *http.Response, err error) (*http.Response, error)
 		attributes[key] = value
 	}
 
-	resources, err := createResources(attributes)
+	resources, err := createResources(res.Request.Context(), attributes, a.projectService)
```

Comment:
With the changes from line 205, you could call `createResources`
```suggestion
	resources, err := a.createResources(res.Request.Context(), attributes)
```